vowpalwabbit/core/src/parse_args.cc

```diff
@@ -332,6 +333,7 @@ void parse_diagnostics(options_i& options, VW::workspace& all)
   if (version_arg)
   {
     std::cout << VW::VERSION.to_string() << " (git commit: " << VW::GIT_COMMIT << ")\n";
+    std::cout << "Enabled features: " << VW::ENABLED_FEATURES << "\n";
```

Comment:
should be more explicit like Compiled features in binary:? Enabled sound like they are on - could cause confusion with flatbuffers that it is in there but not used/enabled

Reply:
yeah that is a good point